src/modules/leaderboard.ts

```diff
@@ -0,0 +1,118 @@
+import { Sequelize, Op } from "sequelize";
+
+import db from "src/db";
+import logger from "src/utils/logger-handler";
+
+async function updateLeaderboardRow(address: string, property: string, value: number) {
+  const userLeaderboard = await db.leaderboard.findOne({
+    where: { address }
+  });
+
+  if(userLeaderboard) {
```

Comment:
add `&& userLeaderboard.hasOwnProperty(property)` so we can protect against funny business and wrong data sent through.